test/main.js

```diff
@@ -666,6 +666,150 @@ describe('glob-stream', function() {
       concat(),
     ], assert);
   });
+
+  it('should handle pattern starting with paren', function(done) {
+    var cwd = dir + '/fixtures/edge';
+    var expected = {
+      cwd: cwd,
+      base: cwd,
+      path: cwd + '/(case)',
+    };
+
+    function assert(pathObjs) {
+      expect(pathObjs.length).toEqual(1);
+      expect(pathObjs[0]).toEqual(expected);
+    }
+
+    pipe([
+      globStream('(case)', { cwd: cwd }),
+      concat(assert),
+    ], done);
+  });
+
+  it('should handle pattern starting with exclam', function(done) {
+    var cwd = dir + '/fixtures/edge';
+    var expected = {
+      cwd: cwd,
+      base: cwd,
+      path: cwd + '/!case',
+    };
+
+    function assert(pathObjs) {
+      expect(pathObjs.length).toEqual(1);
+      expect(pathObjs[0]).toEqual(expected);
+    }
+
+    pipe([
+      globStream('\\!case', { cwd: cwd }),
+      concat(assert),
+    ], done);
+  });
+
+  it('should handle pattern starting with exclam and paren', function(done) {
+    var cwd = dir + '/fixtures/edge';
+    var expected = {
+      cwd: cwd,
+      base: cwd,
+      path: cwd + '/!(case)',
+    };
+
+    function assert(pathObjs) {
+      expect(pathObjs.length).toEqual(1);
+      expect(pathObjs[0]).toEqual(expected);
+    }
+
+    pipe([
+      globStream('\\!(case)', { cwd: cwd }),
+      concat(assert),
+    ], done);
+  });
+
+  it('should handle pattern that is a negative extglob', function(done) {
+    var cwd = dir + '/fixtures/edge';
+
+    function assert(pathObjs) {
+      expect(pathObjs.length).toEqual(3);
+      pathObjs = pathObjs.filter(function(pathObj) {
+        return pathObj.path.substr(cwd.length) === '/case';
+      });
+      expect(pathObjs.length).toEqual(0);
+    }
+
+    pipe([
+      globStream('!(case)', { cwd: cwd }),
+      concat(assert),
+    ], done);
+  });
+
+  it('should ignore pattern starting with paren', function(done) {
+    var cwd = dir + '/fixtures/edge';
+
+    function assert(pathObjs) {
+      expect(pathObjs.length).toEqual(3);
+      pathObjs = pathObjs.filter(function(pathObj) {
+        return pathObj.path.substr(cwd.length) === '/(case)';
+      });
+      expect(pathObjs.length).toEqual(0);
+    }
+
+    pipe([
+      globStream(['*', '!\\(case)'], { cwd: cwd }),
+      concat(assert),
+    ], done);
+  });
+
+  it('should ignore pattern starting with exclam', function(done) {
+    var cwd = dir + '/fixtures/edge';
+
+    function assert(pathObjs) {
+      expect(pathObjs.length).toEqual(3);
+      pathObjs = pathObjs.filter(function(pathObj) {
+        return pathObj.path.substr(cwd.length) === '/!case';
+      });
+      expect(pathObjs.length).toEqual(0);
+    }
+
+    pipe([
+      globStream(['*', '!\\!case'], { cwd: cwd }),
+      concat(assert),
+    ], done);
+  });
+
+  it('should ignore pattern starting with exclam and paren', function(done) {
```

Comment:
and if it's not that, then it's probably how the tests are designed. If node-glob doesn't normalize slashes then wouldn't we need to do double backslashes on windows? Otherwise the current escaping would be recognized as a path separator.